README.md

```diff
@@ -1 +1,93 @@
-# tic-tac-toe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Run on Repl.it](https://repl.it/badge/github/edxco/tic-tac-toe)](https://repl.it/github/edxco/tic-tac-toe)
+# Tic-Tac-Toe
+
+> The famous game: Tic-Tac-Toe
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+- Object Oriented Programming
+
+## Live Demo
+
+[Live Demo Link](https://repl.it/github/edxco/tic-tac-toe)
+
+
+## Getting Started
+ You need 2 players to play the game!
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
Please, add the command to get the local copy
```rb
git clone <repo_url>
```
What is the command the user should type to run it? Kindly add all the needed commands.